Sources/web3swift/Utils/EIP/EIP67Code.swift

```diff
@@ -4,7 +4,9 @@
 //
 
 import Foundation
+#if !os(Linux)
 import CoreImage
+#endif
```

Comment:
We've got to investigate what the hell happening here, and why ERC spec file required `CoreImage` as an import?

Reply:
I believe it's for the creation of the QR code, which is convenient to have on iOS and macOS I assume.

Reply:
I see, so we really could consider just to crop this file for a Linux platform until we figured out how we could implement it properly.